en/includes/references/conditional-auth/api-reference.md

```diff
@@ -723,6 +724,32 @@ This function returns the local user associated with the federate username given
       </tbody>
     </table>
 
+### Get masked value
+
+`getMaskedValue(value)`
+
+This utility function returns a masked value for the given input value. This function can be used to mask sensitive content in adaptive authentication script logs by developers.
```

Comment:
```suggestion
This utility function returns a masked value for the given input value. It can be used to mask sensitive content in adaptive authentication script logs by developers.
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/4664/commits/2fd65e972210c46c6302574ff5035c6cfda5e576